index.html

```diff
@@ -54,17 +54,18 @@ <h1 class="Heading">Vad vi gör</h1>
             <h2 class="Heading Heading--deca">Kontakt och kalas</h2>
           </div>
           <div class="ContactSection-body">
-            <p>Vill ni ha hjälp - hör av er! Vi gör allt ifrån strategi och koncept till digital produktion: video, ljud, copy, kampanjer och större marknadsaktiviteter, alltid anpassat för sociala och digitala kanaler och plattformar. hej@kwdstudio.se</p>
-            <p>Ibland formulerar vi tankar kring varumärken, samtidsfenomen, målgruppsorientering och kalasinbjudningar. Hand som pekar nedåt</p>
+            <p>Vill ni ha hjälp - hör av er! Vi gör allt ifrån strategi och koncept till digital produktion: video, ljud, copy, kampanjer och större marknadsaktiviteter, alltid anpassat för sociala och digitala kanaler och plattformar. 
+            <a href="mailto:hej@kwdstudio.se" itemprop="email">hej@kwdstudio.se</span></a></p>
```

Comment:
Here's a stray `</span>` with no opening `<span>` – probably can just be removed :)